sum.js

```diff
@@ -0,0 +1,9 @@
+const findSum = (array) => {
+    if (array.length === 0) {
+        return 0;
+    } else {
+        return array[0] + findSum(array.slice(1));
+    }
+}
```

Comment:
You must target also array of multiple dimensions because, in question they mentioned **recursively** so also nested arrays must also be considered